dev/io.openliberty.jakarta.interceptor.2.1/bnd.bnd

```diff
@@ -0,0 +1,25 @@
+#*******************************************************************************
+# Copyright (c) 2020, 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jakarta.interceptor.2.1; singleton:=true
+
+
+Export-Package: jakarta.interceptor;version="2.1.0"
+
+instrument.disabled: true
+
+publish.wlp.jar.suffix: dev/api/spec
+
+-buildpath: \
+	jakarta.interceptor:jakarta.interceptor-api;version="[2.1,3)"
```

Comment:
Again weird seeing a range in a buildpath.

Reply:
Again, we discussed and decided that a range is fine but will make it tighter `"[2.1,2.2)"